README.md

```diff
@@ -375,6 +375,7 @@ with GNSS (global navigation satellite system).
 * [OpenCage Geocoding API](https://geocoder.opencagedata.com/) - An API aggregating multiple open geo datasources (OpenStreetMap and others).
 * [OpenMapTiles](https://openmaptiles.com/) - Vector tiles and map services as service, self-hosted or off-line.
 * [Geobox](https://en.geobox.ir/) - Geobox is a suite of products that together cover all stages of building an online GIS.
+* [DataV Atlas](https://atlas.datav.aliyun.com/) - Cloud computing platform for online real-time analysis and visualization of geo bigdata. Powered by AlibabaCloud.
```

Comment:
The link redirects to another website, wich does not look like a Cloud computing platform.

Reply:
Because this platform requires an Alibaba Cloud account to access, so it redirected to the Alibaba Cloud registration page. I think I will change the link to their product intro page, please wait while I change it.